README.md

```diff
@@ -103,19 +103,19 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 0107 | [URI Scheme - Block and transaction objects](./CIP-0107) | Proposed |
 | 0108 | [Governance Metadata - Governance Actions](./CIP-0108) | Proposed |
 | 0109 | [Modular Exponentiation Built-in for Plutus Core](./CIP-0109) | Proposed |
-| 0110 | [Plutus v1 compatible script references](./CIP-0110) | Proposed |
+| 0110 | [Plutus v1 compatible script references](./CIP-0110) | Active |
 | 0112 | [Observe script type](./CIP-0112) | Proposed |
 | 0114 | [CBOR Tags Registry](./CIP-0114) | Proposed |
 | 0115 | [CBOR tag definition - ED25519-BIP32 Keys](./CIP-0115) | Proposed |
 | 0116 | [Universal JSON Encoding for Domain Types](./CIP-0116) | Proposed |
-| 0117 | [Explicit script return values](./CIP-0117) | Proposed |
+| 0117 | [Explicit script return values](./CIP-0117) | Active |
 | 0119 | [Governance Metadata - DReps](./CIP-0119) | Proposed |
 | 0121 | [Integer-ByteString conversions](./CIP-0121) | Active
 | 0122 | [Logical operations over BuiltinByteString](./CIP-0122) | Proposed |
 | 0123 | [Bitwise operations over BuiltinByteString](./CIP-0123) | Proposed |
 | 0128 | [Preserving Order of Transaction Inputs](./CIP-0128) | Proposed |
 | 0381 | [Plutus Support for Pairings Over BLS12-381](./CIP-0381) | Proposed |
-| 1694 | [A proposal for entering the Voltaire phase](./CIP-1694) | Proposed |
+| 1694 | [A proposal for entering the Voltaire phase](./CIP-1694) | Active |
```

Comment:
```suggestion
| 1694 | [A First Step Towards On-Chain Decentralized Governance](./CIP-1694) | Active |
```
fix title

Reply:
@Ryun1 someday this will also be done by GitHub CI, but in the meantime we could use a local "grep" to check _all_ the titles at least once more before then.

Reply:
yup!